x-pack/legacy/plugins/uptime/public/lib/helper/parameterize_values.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const getParameterForList = (key: string, list: string[]) =>
+  list.map(value => `&${key}=${value}`).reduce((prev: string, cur: string) => prev + cur, '');
+
+export const parameterizeValues = (parameters: Record<string, string[]>) =>
+  Object.keys(parameters)
```

Comment:
Like I mentioned in https://github.com/elastic/kibana/pull/50837#discussion_r363733055 the original code that prompted the creation of this function (it wasn't always in a dedicated file) had required parameters, but since this is a general purpose function now that's a really dangerous assumption, so I've modified it to not behave in this way.

Great observation, and I think a smell that the diff grew larger than I'd like as this change progressed.